scripts/portconfig

```diff
@@ -155,6 +159,9 @@ class portconfig(object):
         self.db.mod_entry(PORT_TABLE_NAME, port, {PORT_ADV_SPEEDS_CONFIG_FIELD_NAME: adv_speeds})
 
     def set_interface_type(self, port, interface_type):
+        if self.is_rj45_port:
```

Comment:
Instead of hard coding for rj45 can we have an API which would get interface type from transceiver info and return if setting type is allowed? This would be generically helpful for extending in the future

Reply:
In theory, yes. But I am not sure whether this logic will be extended in the near future so suggest keeping as it is.